### DISCUSSION TITLE
Question

### DISCUSSION THREAD

**Discussion by mcordes92** [eduardogsilva/wireguard_webadmin#25] [General]
Hello, I have to emphasize again how good I think your project is. The implementation is really good and you get it done quickly. I just wanted to ask you casually whether you would be interested in a collaboration. I run my own hosting and we are already using your system on several servers.

**Comment by eduardogsilva** [comment]
Hello @mcordes92 I'm glad to hear that you are using wireguard_webadmin on your hosting provider. 
By the way, did you checked my other project? routerfleet is a tool for managing backups for network equipment. It may be useful for you depending on your environment. 

I'm open to discuss any collaboration on the open source project. It's nice to have people involved and enabling the project to grow faster.

If you need something more particular for you scenario, we can also discuss a project as a freelance or something else. 

Cheers!
